language-server/modules/langserver-core/src/main/java/org/ballerinalang/langserver/completions/providers/context/SelectClauseNodeContext.java

```diff
@@ -64,6 +66,22 @@ public SelectClauseNodeContext() {
         return completionItems;
     }
 
+    @Override
+    public void sort(BallerinaCompletionContext context, SelectClauseNode node, 
+                     List<LSCompletionItem> completionItems) {
+        QueryExpressionNode queryExprNode = (QueryExpressionNode) node.parent();
+
+        for (LSCompletionItem lsCItem : completionItems) {
+            if (SortingUtil.isSymbolCItemWithinNodeAndCursor(context, lsCItem, queryExprNode)) {
+                lsCItem.getCompletionItem().setSortText(SortingUtil.genSortText(1)
+                        + SortingUtil.genSortText(SortingUtil.toRank(context, lsCItem)));
+            } else {
+                lsCItem.getCompletionItem().setSortText(SortingUtil.genSortText(2)
+                        + SortingUtil.genSortText(SortingUtil.toRank(context, lsCItem)));
+            }
```

Comment:
We can reduce the cognitive complexity of this part by assigning the sort text to a separate variable:
```ballerina
	String sortText;
	if (SortingUtil.isSymbolCItemWithinNodeAndCursor(context, lsCItem, queryExprNode)) {
                sorttext = SortingUtil.genSortText(1) + SortingUtil.genSortText(SortingUtil.toRank(context, lsCItem));
       } else {
                sortText = SortingUtil.genSortText(2) + SortingUtil.genSortText(SortingUtil.toRank(context, lsCItem));
       }
            
       lsCItem.getCompletionItem().setSortText(sortText);
```
Same goes to the other files as well.

Reply:
shall we assign the rank to a separate variable and bring the `SortingUtil.genSortText(SortingUtil.toRank(context, lsCItem))` inside the `lsCItem.getCompletionItem().setSortText()` ?